integration_tests/mockApis/nomisMigrationApi.ts

```diff
@@ -424,6 +424,67 @@ const defaultSentencingFailures = {
     },
   ],
 }
+const defaultFailures = {
```

Comment:
made generic - the actual contents really don't matter - so people don't need to copy and paste all the time